### PR TITLE
Add one-liner to get $LATEST linux binary

### PR DESCRIPTION
**Pull request by jbarber**

### CODE REVIEW

File: README.md
Comment (jbarber) [reply README.md]: Sounds good!

If you have a concrete suggestion like this, then please free to use the "Insert a suggestion" Github feature - it cuts down on any ambiguity.

Normally, when it looks like the PR is mostly done and there's some minor feedback but it otherwise looks safe to ship we just approve it and leave comments. This assumes that the PR author isn't going to do the right thing, and means they are free to merge and don't have to wait for another review.

It's up the to the PR author to decide if they have reached the acceptable quality level or need some reassurance before shipping after an earlier review.

Comment (jbarber) [reply README.md]: @CChanHY can you review please!

**Review by ukd1 — APPROVED** [review]

File: README.md
Comment (ukd1) [inline README.md:36]: for extra pretty:

```bash

**Pull request merged**
